README.md

```diff
@@ -53,7 +53,7 @@ nix flake init -t nix-darwin/master
 # To use Nixpkgs 24.11:
 nix flake init -t nix-darwin/nix-darwin-24.11
 
-sed -i '' "s/simple/$(scutil --get LocalHostName)/" flake.nix
+sed -i'' "s/simple/$(scutil --get LocalHostName)/" flake.nix
```

Comment:
```suggestion
sed -i'' -e "s/simple/$(scutil --get LocalHostName)/" flake.nix
```

Reply:
@DamienCassou I think it's still missing the `-e`

Reply:
`-i''` is incorrect for macOS’s default BSD sed. It needs to be `-i ''` or you get a `flake.nix-e` file.